Scribe-i18n/scripts/iOS/convert_xcstrings_to_jsons.py

```diff
@@ -9,17 +9,34 @@
 import json
 import os
 
+# Determine the directory
 directory = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-file = open(os.path.join(directory, "Localizable.xcstrings"), "r").read()
+
+# Read the Localizable.xcstrings file
+try:
+    with open(os.path.join(directory, "Localizable.xcstrings"), "r") as f:
+        file = f.read()
+except FileNotFoundError:
+    print("Error: Localizable.xcstrings file not found.")
+    exit(1)
+
 dir_list = os.listdir(directory)
```

Comment:
I think I missed the fact we restructured the json files into their own directory, it should be `os.path.join(directory, "jsons")` with our current structure? @andrewtavis this could also be an issue with all the other scripts.

Reply:
> I think I missed the fact we restructured the json files into their own directory, it should be `os.path.join(directory, "jsons")` with our current structure? @andrewtavis this could also be an issue with all the other scripts.

I think `os.path.join(directory, "Localizable.xcstrings"` should remain as is or the file `Localizable.xcstrings` to be moved to the folder `jsons` as the file `Localizable.xcstrings` is in the root directory `Scribe-i18n`

Reply:
No let's keep it where it is, as that would complicate Weblate that's reading in the `jsons` directory :)